website/docs/user-guides/uninstall-litmus.md

```diff
@@ -20,15 +20,18 @@ To disconnect the [ChaosAgent](../getting-started/resources#chaosagents) connect
 To remove the respective components of the ChaosAgents you need to manually delete the created resources of that ChaosAgent.
 
 ```bash
-kubectl delete chaosengine <CHAOSENGINE_IDs> --all -A
-kubectl delete chaosexperiments <CHAOSEXPERIMENTS_IDs> --all -A
-kubectl delete chaosresults <CHAOSRESULTS_IDs> --all -A
+kubectl delete chaosengine <CHAOSENGINE_IDs> --all -<AGENT_NAMESPACE>
```

Comment:
If we want to delete by ID as in the instance Id for engines then label sector would be required, or we may update it to engine names and similarly for others like workflow name etc.

Reply:
Updated these to Resource Names